.run/EnrollmentServerWultraApplication.run.xml

```diff
@@ -0,0 +1,11 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="EnrollmentServerWultraApplication" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <option name="ACTIVE_PROFILES" value="dev" />
+    <module name="enrollment-server" />
+    <option name="SPRING_BOOT_MAIN_CLASS" value="com.wultra.app.enrollmentserver.EnrollmentServerApplication" />
+    <option name="VM_PARAMETERS" value="-Dserver.servlet.context-path=/enrollment-server" />
```

Comment:
- [ ] add different port via `server.port=8081` not to be in conflict with powerauth-server